### ISSUE TITLE
Installer fails to start on empty zfs zones

### ISSUE THREAD

**Issue by sebastient** [joyent/smartos-live#459]
Attempting to follow the instructions for building a custom zpool the system fails to boot into the installer after 'zpool destroy zones ; zpool create zones ...' because of missing files.

Should also consider adding the ability to customize zpool creation at install time instead of canned defaults.

https://wiki.smartos.org/display/DOC/SmartOS+Clean+Re-install

**Comment by rmustacc** [comment]
@sebastient can you take a look at the improved installer in https://github.com/joyent/smartos-live/pull/439 and see if the features there will address the desired end state?

**Comment by sebastient** [comment]
Yes just tested #439 and it works like a charm much better installer all around.  Thank you.

**Comment by bahamas10** [comment]
#439 was merged today.. if that fixes this issue feel free to close it, thanks.
